src/haskell/devcontainer-feature.json

```
@@ -0,0 +1,54 @@
+{
+    "id": "haskell",
+    "version": "0.0.1",
+    "name": "Glasgow Haskell Compiler",
+    "documentationURL": "https://github.com/brendandburns/ghc-feature",
```

README files for this repo are auto-generated from `devcontainers-features.json` contents. Can we update this link? Feel free to add any other details to the `NOTES.md` file

```suggestion
    "documentationURL": "https://github.com/devcontainers/features/tree/main/src/haskell",
```

Ref: https://github.com/devcontainers/features/blob/main/src/anaconda/devcontainer-feature.json

---

done.

---

done.